core/tests/puppeteer-acceptance-tests/utilities/user/topic-manager.ts

```diff
@@ -195,6 +199,117 @@ export class TopicManager extends BaseUser {
     await this.clickOn(closeSaveModalButton);
     await this.page.waitForSelector(modalDiv, {hidden: true});
   }
+
+  /**
+   * Open the skill editor page for a skill.
+   */
+  async openSkillEditor(skillName: string): Promise<void> {
+    const skillSelector = this.isViewportAtMobileWidth()
+      ? mobileSkillSelector
+      : desktopSkillSelector;
+    await this.page.bringToFront();
+    await this.navigateToTopicAndSkillsDashboardPage();
+    await this.clickOn(skillsTab);
+    await this.page.waitForSelector(skillSelector, {visible: true});
+
+    await Promise.all([
+      this.page.evaluate(
+        (skillSelector, skillName) => {
+          const skillDivs = Array.from(
+            document.querySelectorAll(skillSelector)
+          );
+          const skillDivToSelect = skillDivs.find(
+            element => element?.textContent.trim() === skillName
+          ) as HTMLElement;
+          if (skillDivToSelect) {
+            skillDivToSelect.click();
+          } else {
+            throw new Error('Cannot open skill editor page.');
+          }
+        },
+        skillSelector,
+        skillName
+      ),
+      this.page.waitForNavigation(),
+    ]);
+  }
+
+  /**
+   * Adds a worked example to the topic.
+   * @param {string} exampleQuestion - The question part of the worked example.
+   * @param {string} exampleExplanation - The explanation part of the worked example.
+   */
+  async addWorkedExample(
+    exampleQuestion: string,
+    exampleExplanation: string
+  ): Promise<void> {
+    const exampleQuestionSelector = '.e2e-test-rte';
```

Comment:
store as constant at the top of the file. Also for the rest.

Reply:
Done.